src/main/java/hudson/plugins/sidebar_link/LinkAction.java

```diff
@@ -59,9 +64,14 @@ public LinkAction(String urlName, String displayName, String iconFileName) throw
             throw new IllegalArgumentException(validationResult);
         }
 
+        if(StringUtils.isBlank(iconFileName)) {
+            iconFileName = "static/efbf17e4/images/16x16/help.png";
```

Comment:
This `efbf17e4` seems to be `SESSION_HASH` from <https://github.com/jenkinsci/jenkins/blob/5661c1055d026424118e2488b84ea22a19bf423d/core/src/main/java/jenkins/model/Jenkins.java#L5330-L5336>. Perhaps the default `iconFileName` should instead be just `"images/16x16/help.png"`. Reading the current hash from `Jenkins.RESOURCE_PATH` seems a worse alternative because the resulting `iconFileName` will be persisted and used after subsequent Jenkins upgrades.

Reply:
Also, `war/src/main/webapp/images/16x16/help.png` was deleted from the Jenkins core in <https://github.com/jenkinsci/jenkins/pull/5778>.

Reply:
Filed <https://github.com/jenkinsci/sidebar-link-plugin/issues/40> about the default icon.